.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,23 @@
+---
+name: 🐞 Bug report
+about: Create a report to help us improve
+---
+
+**Describe the bug**
```

Comment:
Make this headings `## Describe the bug` rather than bold `**Describe the bug**` pls